Automattic/wp-calypso
# Hosting Metrics: Add tooltips to each of our charts

See https://github.com/Automattic/dotcom-forge/issues/3276
See https://github.com/Automattic/dotcom-forge/issues/3274
See https://github.com/Automattic/dotcom-forge/issues/3356

## Proposed Changes

Adds tooltips to each of our charts:

<img width="1384" alt="CleanShot 2023-08-09 at 05 10 14@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/756430b4-368b-45f6-a82e-bc25d6529fdf">

<img width="1380" alt="CleanShot 2023-08-09 at 05 10 27@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/0f6ade39-2d61-4d08-9153-589c403ce029">

<img width="1379" alt="CleanShot 2023-08-09 at 05 10 36@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/b4ab7155-9743-46ab-b2c3-acf561d19d46">

<img width="1393" alt="CleanShot 2023-08-09 at 05 10 47@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/e5f966a5-bde6-4207-98b9-b99e4f0efb40">

## Testing Instructions

1. Navigate to `/site-monitoring/<site>`
2. Verify all of the tooltips appear as expected.
